### ISSUE TITLE
GET Error (Service with the same UUID)

### ISSUE THREAD

**Issue by monza258** [homebridge-plugins/homebridge-camera-ffmpeg#621]
Hi Guys,

Today I get some errors in Homebridge. Does anyone know what's wrong?

```
Jul 07 21:31:10 raspberrypi homebridge[2016]: [7/7/2020, 21:31:10] [Cameras] Initializing Camera-ffmpeg platform...
Jul 07 21:31:10 raspberrypi homebridge[2016]: [7/7/2020, 21:31:10] [Cameras] Configuring accessory Haustür
Jul 07 21:31:10 raspberrypi homebridge[2016]: [7/7/2020, 21:31:10] Error: Cannot add a Service with the same UUID '00000089-0000-1000-8000-0026BB765291' as another Service in this Accessory without also defining a unique 'subtype' property.
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at Accessory._this.addService (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:276:17)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at PlatformAccessory.addService (/usr/local/lib/node_modules/homebridge/src/platformAccessory.ts:92:41)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at FfmpegPlatform.configureAccessory (/usr/local/lib/node_modules/homebridge-camera-ffmpeg/src/index.ts:127:12)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at /usr/local/lib/node_modules/homebridge/src/server.ts:307:25
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at Array.filter (<anonymous>)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at Server.restoreCachedPlatformAccessories (/usr/local/lib/node_modules/homebridge/src/server.ts:270:69)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:158:10)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:80:10)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
Jul 07 21:31:10 raspberrypi homebridge[2016]:     at Module._compile (internal/modules/cjs/loader.js:1138:30)
Jul 07 21:31:10 raspberrypi homebridge[2016]: [7/7/2020, 21:31:10] Got SIGTERM, shutting down Homebridge...
Jul 07 21:31:11 raspberrypi homebridge[2016]: HAP Warning: Characteristic 00000093-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 0000008D-0000-1000-8000-0026BB765291. Adding anyway.
Jul 07 21:31:11 raspberrypi homebridge[2016]: HAP Warning: Characteristic 00000093-0000-1000-8000-0026BB765291 not in required or optional characteristics for service 0000008D-0000-1000-8000-0026BB765291. Adding anyway.
```

**Homebridge Config:**

```
        {
            "name": "Cameras",
            "cameras": [
                {
                    "name": "Haustür",
                    "manufacturer": "FFMPEG",
                    "motion": true,
                    "doorbell": true,
                    "doorbellSwitch": true,
                    "switches": true,
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -re -i rtsp://admin:PASSWORD@192.168.0.42:554/11",
                        "stillImageSource": "-i http://admin:PASSWORD@192.168.0.42/tmpfs/auto.jpg",
                        "maxStreams": 2,
                        "maxWidth": 1280,
                        "maxHeight": 720,
                        "maxFPS": 15,
                        "vcodec": "copy",
                        "audio": true,
                        "debug": false
                    }
                },
                {
                    "name": "Hof",
                    "manufacturer": "FFMPEG",
                    "motion": false,
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -re -i rtsp://admin:PASSWORD@192.168.0.130:554/12",
                        "stillImageSource": "-i http://admin:PASSWORD@192.168.0.130/tmpfs/auto.jpg",
                        "maxStreams": 2,
                        "maxFPS": 15,
                        "vcodec": "copy",
                        "audio": true,
                        "debug": false
                    }
                },
                {
                    "name": "Garten",
                    "manufacturer": "FFMPEG",
                    "motion": false,
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -re -i rtsp://admin:PASSWORD@192.168.0.43:554/12",
                        "stillImageSource": "-i http://admin:PASSWORD@192.168.0.43/tmpfs/auto.jpg",
                        "maxStreams": 2,
                        "maxFPS": 15,
                        "vcodec": "copy",
                        "audio": true,
                        "debug": false
                    }
                },
                {
                    "name": "Terrasse",
                    "manufacturer": "FFMPEG",
                    "motion": false,
                    "videoConfig": {
                        "source": "-rtsp_transport tcp -re -i rtsp://admin:PASSWORD@192.168.0.41:554/12",
                        "stillImageSource": "-i http://admin:PASSWORD@192.168.0.41/tmpfs/auto.jpg",
                        "maxStreams": 2,
                        "maxFPS": 15,
                        "vcodec": "copy",
                        "audio": true,
                        "debug": false
                    }
                }
            ],
            "platform": "Camera-ffmpeg"
```

**Environment:**

* **Node.js Version**:
12.18.1
* **NPM Version**:
6.14.5
* **Homebridge Version**:
1.1.1

* **Homebridge Camera FFmpeg Version**: 
2.1.0
* **Homebridge Config UI X Plugin Version**:
4.23.2
* **Operating System**: 
Raspberry PI

**Comment by donavanbecker** [comment]
Can you test this by installing latest beta
* `sudo npm install -g --unsafe-perm homebridge-camera-ffmpeg@beta`
* To Revert `sudo npm install -g --unsafe-perm homebridge-camera-ffmpeg@latest`

**Comment by monza258** [comment]
Dosnt  work. After reboot Homebridge will kill. 


```
Jul 09 21:30:26 raspberrypi homebridge[800]: [7/9/2020, 21:30:26] Error: Cannot add a Service with the same UUID '00000089-0000-1000-8000-0026BB765291' as another Service in this Accessory without also defining a unique 'subtype' property.
Jul 09 21:30:26 raspberrypi homebridge[800]:     at Accessory._this.addService (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:276:17)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at PlatformAccessory.addService (/usr/local/lib/node_modules/homebridge/src/platformAccessory.ts:92:41)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at FfmpegPlatform.configureAccessory (/usr/local/lib/node_modules/homebridge-camera-ffmpeg/src/index.ts:135:12)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at /usr/local/lib/node_modules/homebridge/src/server.ts:307:25
Jul 09 21:30:26 raspberrypi homebridge[800]:     at Array.filter (<anonymous>)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at Server.restoreCachedPlatformAccessories (/usr/local/lib/node_modules/homebridge/src/server.ts:270:69)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:158:10)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:80:10)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
Jul 09 21:30:26 raspberrypi homebridge[800]:     at Module._compile (internal/modules/cjs/loader.js:1138:30)
Jul 09 21:30:26 raspberrypi homebridge[800]: [7/9/2020, 21:30:26] Got SIGTERM, shutting down Homebridge...
Jul 09 21:30:28 raspberrypi homebridge[800]: [7/9/2020, 21:30:28] [Config] Homebridge Config UI X v4.23.2 is listening on :: port 8080
Jul 09 21:30:31 raspberrypi systemd[1]: homebridge.service: Main process exited, code=exited, status=143/n/a
Jul 09 21:30:31 raspberrypi systemd[1]: homebridge.service: Failed with result 'exit-code'.
```

When i delete accessories folder. Homebridge starts

**Comment by donavanbecker** [comment]
Closing to bring into #631, to see if it fixes your issues.

**Comment by monza258** [comment]
Hi, you closed the issue. Unfortunately, the error is still.

**Comment by Sunoo** [comment]
Given that you’re still having problems, I’ve reopened this issue.

You say that if you delete the accessories folder, the issue clears. I’m guessing if you restart Homebridge again, the issue comes back? Is that correct?

**Comment by monza258** [comment]
> You say that if you delete the accessories folder, the issue clears. I’m guessing if you restart Homebridge again, the issue comes back? Is that correct?

Yes, that's correct. 


I have other messages in the log.
```
Jul 10 21:58:02 raspberrypi systemd[1]: This usually indicates unclean termination of a previous run, or service implementation deficiencies.
Jul 10 21:58:02 raspberrypi systemd[1]: homebridge-camera-ffmpeg.service: Found left-over process 30728 (ffmpeg) in control group while starting unit. Ignoring.
Jul 10 21:58:02 raspberrypi systemd[1]: This usually indicates unclean termination of a previous run, or service implementation deficiencies.
Jul 10 21:58:02 raspberrypi systemd[1]: homebridge-camera-ffmpeg.service: Found left-over process 30842 (ffmpeg) in control group while starting unit. Ignoring.
Jul 10 21:58:02 raspberrypi systemd[1]: This usually indicates unclean termination of a previous run, or service implementation deficiencies.
Jul 10 21:58:02 raspberrypi systemd[1]: homebridge-camera-ffmpeg.service: Found left-over process 31047 (ffmpeg) in control group while starting unit. Ignoring.
Jul 10 21:58:02 raspberrypi systemd[1]: This usually indicates unclean termination of a previous run, or service implementation deficiencies.
Jul 10 21:58:02 raspberrypi systemd[1]: homebridge-camera-ffmpeg.service: Found left-over process 31080 (ffmpeg) in control group while starting unit. Ignoring.
Jul 10 21:58:02 raspberrypi systemd[1]: This usually indicates unclean termination of a previous run, or service implementation deficiencies.
```

**Comment by Sunoo** [comment]
@monza258 Okay, I'm pretty sure I found the issue. The doorbell switch wasn't being created correctly. I've just pushed a fix to the beta branch. If you want to test, you can install with `sudo npm install -g --unsafe-perm homebridge-camera-ffmpeg@beta`

**Comment by monza258** [comment]
Ok, I'm going to test it. Thanks for the quick solution.

**Comment by Sunoo** [comment]
Should be fixed in 2.2.0, reopen if this is still occurring.
